app/controllers/api/databases.php

```diff
@@ -193,16 +193,14 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
         ->setType(DATABASE_TYPE_CREATE_ATTRIBUTE)
         ->setDatabase($db)
         ->setCollection($collection)
-        ->setDocument($attribute)
-    ;
+        ->setDocument($attribute);
```

Comment:
Please remove all the formatting updates, it makes the PR harder to review as it adds a lot of unrelated changes to the diff